src/security/cli_security.c

```diff
@@ -33,6 +33,7 @@ dc_sec_request_creds(d_iov_t *creds)
 
 	rc = request_credentials_via_drpc(&response);
 	if (rc != DER_SUCCESS) {
+		drpc_response_free(response);
```

Comment:
Doesn't hurt anything, but I suspect what you actually ran into here was an issue in a test, not an issue in the production code. In practice drpc_call shouldn't be allocating a response if it returned a failure. But at this level we're mocking drpc_call in the unit tests, so the test probably didn't clean up the mock.